internal/fnruntime/runner.go

```diff
@@ -286,13 +324,13 @@ func enforcePathInvariants(nodes []*yaml.RNode) error {
 		if strings.HasPrefix(fp, "../") {
 			return fmt.Errorf("function must not modify resources outside of package: resource has path %s", currPath)
```

Comment:
should this error message include pkgPath as well?

Reply:
This error message is printed just below the package path in the usual render progress reporting, so it's not needed.